redis/src/main/scala/zio/redis/api/SortedSets.scala

```diff
@@ -842,22 +897,24 @@ trait SortedSets {
    * @return
    *   Returns the items for this iteration or nothing when you reach the end.
    */
-  final def zScan[K: Schema, M: Schema](
+  final def zScan[K: Schema](
     key: K,
     cursor: Long,
     pattern: Option[String] = None,
     count: Option[Count] = None
-  ): ZIO[RedisExecutor, RedisError, (Long, Chunk[MemberScore[M]])] = {
-    val memberScoresOutput =
-      ChunkTuple2Output(ArbitraryOutput[M](), DoubleOutput)
-        .map(_.map { case (m, s) => MemberScore(s, m) })
-    val command = RedisCommand(
-      ZScan,
-      Tuple4(ArbitraryInput[K](), LongInput, OptionalInput(PatternInput), OptionalInput(CountInput)),
-      Tuple2Output(MultiStringOutput.map(_.toLong), memberScoresOutput)
-    )
-    command.run((key, cursor, pattern.map(Pattern), count))
-  }
+  ): ResultBuilder1[({ type lambda[x] = (Long, MemberScores[x]) })#lambda] =
```

Comment:
Can you create a type alias instead of using type lambda?

Reply:
Same here.